src/comments.ts

```diff
@@ -59,7 +57,7 @@ ${names.map(n => `${n}`).join(" ")}
 });
 
 export const PingStaleReviewer = (reviewedAbbrOid: string, reviewers: string[]) => ({
-    tag: `stale-ping-${sha256(reviewers.join("-")).substr(0, 6)}-${reviewedAbbrOid}`,
+    tag: `stale-ping-${reviewedAbbrOid}`,
```

Comment:
Why are you dropping the reviewers' hash too?

Reply:
A change in `reviewers` implies a change in `reviewedAbbrOid`, which the tag already contains, because the only way to change the stale reviewers is for a new review to become stale.